account_prof_menu_page.php

```diff
@@ -151,12 +151,12 @@
 ?>
 								<td class="center">
 									<?php if( $t_is_global_profile ) { ?>
-									<i class="ace-icon fa fa-check fa-lg"></i>
+									<?php print_icon( 'fa-check', 'ace-icon fa-lg" ); ?>
```

Comment:
Syntax error
`'ace-icon fa-lg"` should be `'ace-icon fa-lg'`

Reply:
oops